.github/workflows/periodic-cleanup.yaml

```
@@ -19,27 +23,50 @@ jobs:
+      KONFLUX_API_SERVER: ${{ secrets.KONFLUX_PUBLIC_STAGE_API_SERVER }}
+      KONFLUX_TOKEN: ${{ secrets.KONFLUX_INTEGRATION_CLEANER }}
+      KONFLUX_NAMESPACE: ${{ secrets.KONFLUX_INTEGRATION_NAMESPACE }}
+      # Dry run mode (only applies to Konflux cleanup)
+      DRY_RUN: ${{ github.event_name == 'schedule' && 'false' || github.event.inputs.dry_run }}
       JOB_TYPE: 'periodic-cleanup'
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        # Checkout specific branch
+        with:
+          ref: ${{ github.event.inputs.ref }}
```

actionlint (1.7.11)

[error] property "ref" is not defined in object type {dry_run: string}

(expression)